python/cugraph/structure/graph.py

```diff
@@ -171,191 +171,35 @@ def view_edge_list(self):
             The expected type of the gdf_column element is floating point
             number.
         """
-        source_col, dest_col, value_col = \
-            graph_wrapper.view_edge_list(self.graph_ptr)
-
-        return source_col, dest_col, value_col
+        if self.edgelist is None:
+            graph_wrapper.view_edge_list(self)
+        return self.edgelist.edgelist_df
 
     def delete_edge_list(self):
         """
         Delete the edge list.
         """
-        graph_wrapper.delete_edge_list(self.graph_ptr)
-
         # decrease reference count to free memory if the referenced objects are
         # no longer used.
-        self.edge_list_source_col = None
-        self.edge_list_dest_col = None
-        self.edge_list_value_col = None
-
-    def add_adj_list(self, offset_col, index_col, value_col=None, copy=False):
-        """
```

Comment:
I think documentation got accidentally deleted

Reply:
yes, i would be adding.updating documentation

Reply:
updated